### ISSUE TITLE
Dual player probably only from iPhone

### ISSUE THREAD

**Issue by Teygeta** [vidstack/player#964]
### Current Behavior:
Through the use of the code written below, I end up with two overlapping players (see photo below) when I try to watch a video from an Apple iPhone phone.
```
<media-player
    autoplay
    playsinline
    :src="src"
  >
    <media-provider 
    autoplay
    playsinline />
    <media-video-layout />
  </media-player>
```
It is as if the Vidstack player is overlaid on top of the "default" player on the phone.

### Expected Behavior:
I expect that the phone player will not be shown at the time the Vidstack player is present 

### Steps To Reproduce:
Use my code shown above on an Apple iPhone phone (I have not tried with ipad)

### Environment:
- Framework: Vue
- Meta Framework: Nuxt.js
- Node: 18.0.0
- Device: iPhone
- OS: iOS@17
- Browser: Chrome

### Photo:

![image](https://github.com/vidstack/player/assets/98349901/a678f43c-e417-4a33-bb34-167288f3310d)

**Comment by mihar-22** [comment]
I've seen this before but I believe this is a localhost bug. There is no fullscreen API available on iOS Safari (iPhone), we can't fullscreen the custom UI. I don't know how it's possible. We can add some styles to controls to hide it in this case.

**Comment by mihar-22** [comment]
Should be fixed in `1.4.0`. If not we can reopen and dig into it. I couldn't repro, this bug is unfortunately random as hell.

**Comment by Teygeta** [comment]
I am using cludflare stream

**Comment by mihar-22** [comment]
Figured out why it was incorrectly inferring live stream. The iPhone bug is super weird, I'll see if I can find out what's up. The native UI should not be displayed if `playsinline` is set.

**Comment by mihar-22** [comment]
I tested this on iOS@17 on both my phone and the simulator. I thought it mighe be an issue with `playsinline` not being set in Nuxt but that's not the case either.

Try restarting your browser on your phone and seeing if it's resolved. I need a repro of you current setup. As it stands I can't confirm anything.

**Comment by BlakeB415** [comment]
I am also experiencing this issue with Nuxt.

**Comment by mihar-22** [comment]
I tested it and it was working. I'm not sure why Nuxt is not setting it in your case? This might be a bug to bring up with them. If I can get a full repro I'll look more into it.

**Comment by BlakeB415** [comment]
> king. I'm not sure why Nuxt is not setting it in your case? This might be a bug to bring up with them. If I can get a full repro I'll look more into it.

Seems to only be an issue with HLS

**Comment by BlakeB415** [comment]
Here is a reproduction. 
https://stackblitz.com/edit/github-ndtlmv

I needed to wrap the player in <ClientOnly> to force it to load client side.

**Comment by mihar-22** [comment]
Strangely setting it as `<media-player playsInline>` seems to work. I'm not sure why the attribute is not by Nuxt.

**Comment by mihar-22** [comment]
Okay I realized today that `playsInline` is the property on `HTMLVideoElement` so some frameworks attempt to set that property. I'm pushing a fix out which should resolve this in `1.4.8`.

There's another issue that setting `playsinline` late in iOS doesn't seem to take effect. It has to be set before the `src` is set, otherwise it won't work.
